hypha/apply/funds/templates/funds/includes/co-applicant-block.html

```diff
@@ -0,0 +1,60 @@
+{% load i18n heroicons %}
+
+<div class="sidebar__inner sidebar__inner--actions" data-testid="sidebar-primary-actions">
+    <details {% if user.is_applicant %} open {% endif %}>
+        <summary class="flex gap-4 justify-between items-center">
+            <p class="flex items-center text-lg font-bold">
+                {% trans "Co-applicants" %}
+                {% heroicon_solid "chevron-down" size="16" class="w-4 h-4 ms-2" %}
+            </p>
+            <button
+                class="flex items-center py-1 pr-4 pl-2 font-bold rounded-sm -me-2 {% if object.co_applicant_invites.count >= invite_max_limit %} text-blue-200 {% else %}transition-colors cursor-pointer text-dark-blue hover:bg-slate-200{% endif %}"
+                hx-get="{% url 'apply:submissions:invite_co_applicant' pk=object.id %}"
+                hx-target="#htmx-modal"
+                {% if object.co_applicant_invites.count >= invite_max_limit %}disabled title='{% trans "Max limit reached" %}'{% endif %}
+                role="button"
+                aria-label="{% trans "Invite CoApplicant" %}"
```

Comment:
User facing text should say "co-applicant".

Reply:
updated